docs/search_dsl.rst

```diff
@@ -218,6 +218,24 @@ acts as a top-level aggregation:
 .. code:: python
 
     s = Search()
+    a = A('terms', field='category')
+    s.aggs.metric('category_terms', a)
```

Comment:
If you are adding a `terms` aggregation, it should be added as `bucket`, not metric.